### DISCUSSION TITLE
Got an unexpected keyword arg 'class_names' #1019

### DISCUSSION THREAD

**Discussion by MananShukla7** [mrdbourke/pytorch-deep-learning#325] [Q&A] [answered]
Hello,
I am trying to plot a confusion matrix with class names but its giving me
"TypeError: plot_confusion_matrix() got an unexpected keyword argument 'class_names''

**code:**
import mlxtend
from mlxtend.plotting import plot_confusion_matrix
fig,ax=plot_confusion_matrix(conf_m.numpy(),
class_names=class_names,
figsize=(10,7))
plt.show();

I am working on FashionMNIST dataset

Below is the image of the error:
![image](https://user-images.githubusercontent.com/68244767/222884691-a8134221-dd52-415b-a20c-9e0162572bde.png)

**Comment by karthikbhandary2** [comment]
which section is this from?

**Comment by MananShukla7** [comment]
It's from 04-Comp Vision 
vid number-123. Evaluating model predictions with a confusion matrix

**Comment by MananShukla7** [comment, marked as answer]
Apparently, I was using the older mlxtend version

![image](https://user-images.githubusercontent.com/68244767/222968341-717a61c3-3a31-441c-9762-a537ea8eb092.png)

The most current version is 0.21.0 (from Sep 2022)

Use this command:

`!pip3 install --upgrade mlxtend` 
and then restart runtime to upgrade to newer version.

![image](https://user-images.githubusercontent.com/68244767/222968909-69485def-d74c-48e8-bf7c-f45229e0845b.png)

**Reply by mrdbourke** [reply]
Glad to see you got it fixed! Thank you for adding an answer

**Reply by MananShukla7** [reply]
Thanks Daniel, I'm glad that i could be of help !
